packages/fxa-auth-server/config/index.ts

```diff
@@ -3,11 +3,13 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 import convict from 'convict';
 import fs from 'fs';
-import { makeMySQLConfig, makeRedisConfig } from 'fxa-shared/db/config';
+import { makeRedisConfig } from 'fxa-shared/db/config';
 import { tracingConfig } from 'fxa-shared/tracing/config';
 import path from 'path';
 import url from 'url';
 
+import { makeConvictMySQLConfig as makeMySQLConfig } from '../../../libs/shared/db/mysql/core/src/lib/config';
```

Comment:
Do we want to use relative paths? I might be mistaken but I thought nx [provided a way](https://nx.dev/more-concepts/applications-and-libraries#applications-and-libraries) to reference libs.

Reply:
I've updated this, but I do remember there being a conversation around this possibly causing issues.

@julianpoy I believe this was something you might have mentioned?

Reply:
Ah right, [this is an example](https://app.circleci.com/pipelines/github/mozilla/fxa/42871/workflows/c8e456e8-adeb-4e03-a38f-2866de11af4e/jobs/451228/parallel-runs/0/steps/0-106) of the errors. I'll revert the change.

Is there a plan to fix this in the future?